blocks/text-output-block/output-types/.npmignore

```diff
@@ -0,0 +1 @@
+*.test.jsx
```

Comment:
```suggestion
*.test.jsx

```

Reply:
minor nitpick, blank EOL recommended

Reply:
@nschubach Fixed it in [eb27f54](https://github.com/WPMedia/feed-components/pull/654/commits/eb27f54e7678040ec81acdb4308002c9c719b2b0)